mycroft/client/speech/mic.py

```diff
@@ -400,6 +405,10 @@ def _upload_wake_word(self, audio, metadata):
             }
         )
 
+    def trigger_listen(self):
```

Comment:
Probably good to have a log here to note when skipping the wake word loop is manually triggered. Here of in _skip_wake_word I guess.

Reply:
Added debug level log message

Reply:
:+1: